# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webidl/TransceiverImpl.webidl
# Commit: 0b03dd9d20ac
# Full Hash: 0b03dd9d20ace235502714101097fe34d1864563
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2019-07-31 15:46:43
# Regressor Bug: 1493613
# File Overlap Count: 1
# Description:
#   Bug 1493613 - Update muted state through MediaStreamTrackSource. r=bwc,smaug
#   
#   This ensures all clones of the original track also receives the new muted state.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D37933
# ==============================================================================

diff -r cf84eeffed22 -r 0b03dd9d20ac dom/webidl/TransceiverImpl.webidl
--- a/dom/webidl/TransceiverImpl.webidl	Tue Jul 30 14:55:48 2019 +0000
+++ b/dom/webidl/TransceiverImpl.webidl	Wed Jul 31 09:35:06 2019 +0000
@@ -17,6 +17,7 @@
 [ChromeOnly]
 interface TransceiverImpl {
   MediaStreamTrack getReceiveTrack();
+  void setReceiveTrackMuted(boolean muted);
   [Throws]
   void syncWithJS(RTCRtpTransceiver transceiver);
 };